src/client/dfuse/pil4dfs/int_dfs.c

```diff
@@ -599,19 +598,19 @@ discover_daos_mount(void)
 
 	pool = getenv("DAOS_POOL");
 	if (pool == NULL) {
-		D_WARN("warning> DAOS_POOL is not set.\n");
+		D_WARN("DAOS_POOL is not set.");
```

Comment:
This and others should have a newline.
```suggestion
		D_WARN("DAOS_POOL is not set.\n");
```
Is there any documentation for using pil4dfs with the DAOS_MOUNT_POINT and other environment varliables?

In addition there's the duns_resolve_path() function that can convert from a path to a pool/container that would be useful here, can you file a ticket to support that please.

Reply:
We do not have document for libpil4dfs yet. We will add it later.
Yes. "duns_resolve_path()" is called inside "is_path_start_with_daos()". It is on to-do list. A new ticket was created for this task.
https://daosio.atlassian.net/browse/DAOS-13524